test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -253,6 +354,13 @@ if [ "${wt_standalone}" = true ]; then
     (build_branch "$wt2")
 fi
 
+if [ "$newer" = true ]; then
+    create_configs_per_newer_release
+else
+    create_configs_older_standalone_release
+fi
```

Comment:
the `newer` and `older` variables are used to distinguish among MongoDB (non-standalone) release branches. I found a bit confusing seeing the check here to call the new `create_configs_older_standalone_release` function.

Reply:
And can we merge the calling of those new create_configs functions into below `$newer/$older/$wt_standalone` checking logics if possible?